.gitattributes

```diff
@@ -41,3 +41,14 @@
 
 # Custom for Visual Studio
 *.cs     diff=csharp
+
+
+# Files excluded from git packages
+tests/ export-ignore
```

Comment:
For some reasons we also allow to have tests in `Test` directory which we should remove as well if these paths apply to nested directories as well? or do they only apply to directories in root directory? Maybe we need to define `/tests` or something similar to .gitignore?